packages/common-scripts/index.d.ts

```diff
@@ -0,0 +1,85 @@
+import { TransactionSkeleton } from "@ckb-lumos/types"
+
+// TODO: secp256k1Blake160 types
+
+export declare const secp256k1Blake160Multisig: {
+  /**
+   * transfer capacity from multisig script cells
+   *
+   * @param txSkeleton
+   * @param fromAddress must be multisig script address
+   * @param fromInfo multisig parameters
+   * @param toAddress can be any type of lock script and can left empty
+   * @param amount transfer CKB capacity in shannon
+   * @param options
+   */
+  transfer(
+    txSkeleton: TransactionSkeleton,
+    fromAddress: string,
+    fromInfo: {
+      R: number,
+      M: number,
+      publicKeyHashes: string[],
+    },
+    toAddress: string | undefined,
+    amount: string | bigint,
+    options: {
+      config: any, // TODO: define this type later
```

Comment:
We can define a Config type now which is just any, that will make future changes easier

Reply:
updated